github-release/github-release.sh

```diff
@@ -48,14 +48,23 @@ add_file_to_release() {
     echo "Posting binary contents of $1 to $CURL_URL"
 
     if [ "$DRY_RUN" = "false" ] ; then
-        curl -X POST \
+        http_code=$(curl -X POST \
             -H "Authorization: Bearer $GITHUB_TOKEN" \
             -H "Accept: application/vnd.github+json" \
             -H "X-GitHub-Api-Version: 2022-11-28" \
             -H "Content-Type: application/octet-stream" \
-            --fail \
```

Comment:
We're handling the HTTP code ourselves now, so no `--fail`; but we are capturing stdout now because it contains the HTTP code, so `--silent`.